<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iOS Style Qt Application</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>16</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QFrame" name="headerCard">
      <layout class="QVBoxLayout" name="headerLayout">
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="text">
          <string>App Dashboard</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="subtitleLabel">
         <property name="text">
          <string>Navigate through different app sections</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>16</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QFrame" name="card1">
          <layout class="QVBoxLayout" name="card1Layout">
           <item>
            <widget class="QLabel" name="description1">
             <property name="text">
              <string>Essential system apps and settings</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton1_1">
             <property name="text">
              <string>üì± Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton1_2">
             <property name="text">
              <string>üìß Messages</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton1_3">
             <property name="text">
              <string>üìû Phone</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>Apps</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>16</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QFrame" name="card2">
          <layout class="QVBoxLayout" name="card2Layout">
           <item>
            <widget class="QLabel" name="description2">
             <property name="text">
              <string>Entertainment and media applications</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton2_1">
             <property name="text">
              <string>üéµ Music</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton2_2">
             <property name="text">
              <string>üì∑ Camera</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton2_3">
             <property name="text">
              <string>üåê Safari</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="title">
        <string>Tools</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>16</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <widget class="QFrame" name="card3">
          <layout class="QVBoxLayout" name="card3Layout">
           <item>
            <widget class="QLabel" name="description3">
             <property name="text">
              <string>Utility and productivity tools</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton3_1">
             <property name="text">
              <string>üìä Calculator</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton3_2">
             <property name="text">
              <string>‚è∞ Clock</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton3_3">
             <property name="text">
              <string>üóÇÔ∏è Files</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="visible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="visible">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
